packages/stream/solana/StreamClient.ts

```diff
@@ -475,7 +601,7 @@ export class SolanaStreamClient extends BaseStreamClient {
 
     for (const { ixs, metadata, recipient } of instructionsBatch) {
       const messageV0 = new TransactionMessage({
-        payerKey: sender.publicKey,
+        payerKey: sender.publicKey!,
```

Comment:
unsafe

Reply:
duplicated multiple times because of removing the check above

Reply:
`sender.publicKey` is checked for in the call of `prepareCreateInstruction` so `sender.PublicKey` can't be undefined here as it would have thrown an error. 

I should be checking for `recipient.length` though as if that is 0 this will get called without `prepareCreateInstructions` being called

Reply:
But I understand your concern, and it's safer to have the check in this function too, so I will add it

Reply:
my comment above about removing the check. 
either - put explicit check in both places or make it obvious with function signature (that sender.publicKey couldn't be nullable)